packages/navigator/package.json

```diff
@@ -44,6 +43,7 @@
     "react-dom": "~16.9.0"
   },
   "devDependencies": {
+    "@jupyterlab/buildutils": "^2.0.0",
```

Comment:
Looks like we might be able to drop the dependency if it's not used?

https://github.com/mamba-org/gator/blob/bea611ea5d508b9da8e240cd736ae59e3d24fa75/packages/navigator/webpack.config.js#L6

Reply:
Thanks for catching that. I'll clean the code.